index.js

```diff
@@ -154,6 +154,16 @@ module.exports = function (host, port) {
       return send('add', null, null, file, cb)
     },
     cat: argCommand('cat'),
+    slurp: function (path, cb) {
+      this.cat(path, function (err, stream) {
+        if (err) return cb(err)
+        var buf = ''
+        stream
+          .on('error', cb)
+          .on('data', function (data) { buf += data })
+          .on('end', function () { cb(null, buf) })
+      })
+    },
```

Comment:
what i'm worried about is dedicating a whole command in the top level api for this. people will expect it elsewhere. i want to be able to claim that the API looks the same _everywhere_ (node-ipfs-api, the actual http api on go-ipfs, the cli, etc)

i'd define it like `_slurp` or something.